index.js

```diff
@@ -77,7 +77,7 @@ module.exports = {
         type: 'input',
         name: 'issues',
         message: dedent`
-          List any issues closed by this change:
+          List any issue(s) closed by this change (e.g. #123, #124):
```

Comment:
How about omming the hash and let the user submit a list of numbers? (e.g. `123, 124, 125`)